docs/extensions/extensions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Phylum CLI Extensions
+category: TODO
```

Comment:
@furi0us333 I think it makes sense to put the documentation for writing extensions in a different section. You can generate a new category ID for that once this is settled, right?

Reply:
I added a new category in the docs, please use `62c5cb137dbdad00536291a6` to ensure relevant documentation lands under the correct category.